src/clr-angular/forms/select/select.spec.ts

```diff
@@ -139,5 +162,18 @@ export default function(): void {
                 expect(context.clarityDirective.closeMenuOnTabPress).toHaveBeenCalled();
             });
         });
+
+        describe("Select with Menu", () => {
+            beforeEach(function() {
+                context = this.create(ClrSelect, TestSelectWithMenu, [IfOpenService]);
+                ifOpenService = context.getClarityProvider(IfOpenService);
+            });
+
+            it("renders the menu projected by the consumer", () => {
+                const menus = context.clarityElement.querySelectorAll("clr-options");
+                expect(menus.length).toBe(1);
+                expect(menus[0].classList.contains("test")).toBe(true);
```

Comment:
I like this sneaky class check to make sure we're not cheating. Awesome. 👍

Reply:
thanks!